extensions/objects/models/objects.py

```diff
@@ -623,6 +623,8 @@ class NormalizedRectangle2D(BaseObject):
     def normalize(cls, raw):
         # Moves cur_value to the nearest available value in the range
         # [min_value, max_value].
+        """Returns the normalized coordinates of the rectangle.
```

Comment:
This isn't a `@property`, so add `Args:`, `Returns:` docstrings.

Reply:
Done.